wp-content/plugins/automatic-translation/TranslationManager.php

```diff
@@ -121,6 +139,15 @@ private function create_translation($post, $source_language_code, $target_langua
 		$this->add_save_post_hook();
 		$this->wpml_helper->link_wpml($post->ID, $post->post_type, $translated_post_id, $target_language_code);
 		$this->mark_as_automatic_translation($translated_post_id);
+		
+		//fixes wrong post_parent for autotranslation
```

Comment:
With calling $this->wpml_helper->get_translation_post_parent one wp_update_post is enough.